charm/src/charm.py

```diff
@@ -56,6 +59,14 @@ def version(self) -> str:
         # TODO: get the version somehow and return it
         return "Version ?"
 
+    def _require_nginx_route(self):
+        require_nginx_route(
+            charm=self,
+            service_hostname=self.config["external_hostname"],
+            service_name=self.app.name,
+            service_port=5000,
```

Comment:
I considered that, but I couldn't come up with a good reason why it should be an extra config item since it's only the internal port. Is there ever a need to change this that you can think of?